MSB4100 error when a project has no PackageReference elements

When one of the project KoreBuild is trying to build has no PackageReference elements, you end up with this error:

```
error MSB4100: Expected "$(_ReferencesInternalAspNetCoreSdk)" to evaluate to a boolean instead of "", in condition "'$(IncludeSymbols)' == '' AND $(_ReferencesInternalAspNetCoreSdk)".
```

This occurs because `_ReferencesInternalAspNetCoreSdk` is defined by searching for a PackageReference with the Identity value `Internal.AspNetCore.Sdk` but when there are no PackageReference elements the MSBuild expression evaluates to `''` instead of `'false'`

The problematic expression in question:

https://github.com/aspnet/BuildTools/blob/ad09c8008e2f4ad63757131dd8c5a8e8a7653d60/files/KoreBuild/modules/solutionbuild/Project.Inspection.targets#L18-L19

Because `_ReferencesInternalAspNetCoreSdk` is `''`, it fails to function as a boolean.

PR to fix coming soon.

--- a/files/KoreBuild/modules/solutionbuild/Project.Inspection.targets
+++ b/files/KoreBuild/modules/solutionbuild/Project.Inspection.targets
@@ -10,12 +10,16 @@
       <NormalizedPackageVersion Condition="$(NormalizedPackageVersion.Contains('+'))">$(PackageVersion.Substring(0, $(PackageVersion.IndexOf('+'))))</NormalizedPackageVersion>
       <FullPackageOutputPath>$(PackageOutputPath)$(PackageId).$(NormalizedPackageVersion).nupkg</FullPackageOutputPath>
       <SymbolsPackageOutputPath>$(PackageOutputPath)$(PackageId).$(NormalizedPackageVersion).symbols.nupkg</SymbolsPackageOutputPath>
+      
+      <!-- Detect if Internal.AspNetCore.Sdk is referenced -->
+      <_ReferencesInternalAspNetCoreSdk Condition="'$(_ReferencesInternalAspNetCoreSdk)' == ''">@(PackageReference->AnyHaveMetadataValue('Identity', 'Internal.AspNetCore.Sdk'))</_ReferencesInternalAspNetCoreSdk>
+      <!-- If _ReferencesInternalAspNetCoreSdk is still empty, it means there are no package references, so it should be false -->
+      <_ReferencesInternalAspNetCoreSdk Condition="'$(_ReferencesInternalAspNetCoreSdk)' == ''">false</_ReferencesInternalAspNetCoreSdk>
 
       <!--
         Workaround for design time builds on projects using Internal.AspNetCore.Sdk that have not yet been restored.
         IncludeSymbols and IncludeSource are off by default but are enabled in Internal.AspNetCore.Sdk.
       -->
-      <_ReferencesInternalAspNetCoreSdk Condition="'$(_ReferencesInternalAspNetCoreSdk)' == ''">@(PackageReference->AnyHaveMetadataValue('Identity', 'Internal.AspNetCore.Sdk'))</_ReferencesInternalAspNetCoreSdk>
       <IncludeSymbols Condition="'$(IncludeSymbols)' == '' AND $(_ReferencesInternalAspNetCoreSdk)">true</IncludeSymbols>
       <IncludeSource Condition="'$(IncludeSource)' == '' AND $(_ReferencesInternalAspNetCoreSdk)">true</IncludeSource>
     </PropertyGroup>

